### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by alex-fitzgerald** [facebook/stylex#678] [General]
Just wanted to drop a note about how superb the `stylex` package is; how clear and comprehensive the documentation, how engaged the community engagement is - I've been blown away with how thorough and helpful answers to others' questions have been.

I've been proof of concepting moving a quite expansive project from `styled-components` to StyleX, and not only has the tool been such a dream to work with, any questions I have had have either been so well answered either in the documentation or in the Q&A section it's made my entire project a breeze.

You guys are **_stylexcellent_**

**Comment by nmn** [comment]
Thank you @alex-fitzgerald! It means a lot.
